airflow/upload-to-environment.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+for filename in dags/*; do
+  echo "Uploading Dag: ${filename}"
+    gcloud composer environments storage dags import --environment data-ingestion-environment --source $filename --location=us-central1
```

Comment:
Does this location have to be explicitly specified?

Reply:
Yeah it's a required flag.

https://cloud.google.com/sdk/gcloud/reference/composer/environments/storage/dags/import